File: tests/clapi/functional/resources/code/php/example/src/Course/Course.php
<?php

declare(strict_types=1);

namespace App;

use App\Shared\AggregateRoot;
use App\Shared\SomeTrait;

final class Course extends AggregateRoot
{
    use SomeTrait;

    private $att = [];

    public function __construct(private readonly int $id, private string $name, private readonly float $duration)
    {
    }

    public function foo(): void
    {
    }

    private function bar(): string
    {
        return "bar";
    }

    public function fizz(): int
    {
        return 0;

    }

}

File: tests/clapi/functional/resources/code/java/example/src/com/example/course/Course.java
package com.example.course;

import com.example.shared.AggregateRoot;
import java.util.HashMap;
import java.util.Map;

public final class Course extends AggregateRoot {
    private final int id;
    private String name;
    private final float duration;
    private final Map<String, Object> att = new HashMap<>();

    public Course(int id, String name, float duration) {
        this.id = id;
        this.name = name;
        this.duration = duration;
    }

    public void foo() {
    }

    private String bar() {
        return "bar";
    }

    public int fizz() {
        return 0;
    }
}
